kuttl-test.yaml

```diff
@@ -3,10 +3,8 @@ kind: TestSuite
 manifestDirs:
 - ./test/manifests/
 commands:
-  - command: ./bin/kubectl-kudo init --crd-only
-  - command: ./bin/manager
-    background: true
-  - command: sleep 10
```

Comment:
I had to replace `sleep`s with waiting for `kudo-manager-pod` `status.phase == Running` because while for some reason, k8s 1.16 tests needed 30s sleep, for k8s 1.7 it was 60s (and I still wasn't sure that it wouldn't be flaky).